cmake/modules/LXQtTranslateTs.cmake

```diff
@@ -84,7 +92,66 @@ function(lxqt_translate_ts qmFiles)
         set(TR_TRANSLATION_DIR "translations")
     endif()
 
-    file(GLOB tsFiles "${TR_TRANSLATION_DIR}/${TR_TEMPLATE}_*.ts")
+    if (NOT DEFINED LXQT_TRANSLATIONS_REPO)
+        set(LXQT_TRANSLATIONS_REPO "https://github.com/palinek/translations.git")
```

Comment:
Of course not... it's only a testing one, because I'm lacking write permission in the newly created https://github.com/lxde/translations